sdk/lusid/utilities/refreshing_token.py

```diff
@@ -0,0 +1,55 @@
+import requests
+import base64
+
+from datetime import datetime
+from datetime import timedelta
+from collections import UserString
+
+
+class RefreshingToken(UserString):
+
+    def __init__(self, token_url, client_id, client_secret, access_token, refresh_token, expiry_offset=60):
+
+        token_data = {
+            "expires": datetime.utcnow(),
```

Comment:
I think if you replace this with the expiry inside access_token it won't run straight away as John suggested.

Reply:
done

Reply:
fixed